test/unit_tests/braket/experimental/autoqasm/test_transpiler.py

```diff
@@ -71,50 +68,74 @@ def bell(q0: int, q1: int):
         h(q0)
         cnot(q0, q1)
 
-    @aq.main
-    def bell_decorated(q0: int, q1: int):
-        bell(q0, q1)
+    expected_partial = """OPENQASM 3.0;
+input int[32] q1;
+qubit[2] __qubits__;
+h __qubits__[1];
+cnot __qubits__[1], __qubits__[q1];"""
 
-    expected = """OPENQASM 3.0;
-qubit[4] __qubits__;
+    expected_no_arg_partial = """OPENQASM 3.0;
+qubit[2] __qubits__;
```

Comment:
The qubit length needs to be max qubit index + 1

```suggestion
qubit[4] __qubits__;
```

Reply:
Do we have any plans to validate this? This scenario seems like an example of when we have sufficient information to assert this